.gitignore

```diff
@@ -4,7 +4,7 @@
 *.dll
 *.so
 *.dylib
-ktf*
+ktf.*.*
```

Comment:
This feels hacky

Suggestion, optional: put output binaries in a `build/` directory or similar, and gitignore it as a whole.

Reply:
I don't know about "hacky" as it's following the build pattern for artifacts, but I made the requested change anyway: 1f4f09aab5adadcc48e8fd9e79b2fef948c4dede